### PR TITLE
Home

### PR DESCRIPTION
**Pull request by nitindhimangithub**
header,footer,banner section ,new arrival section category section

### CODE REVIEW

File: src/features/Banner/Banner.js
Comment (rimple0) [inline src/features/Banner/Banner.js:1]: add alt for image

File: src/features/Footer/Footer.js
Comment (rimple0) [inline src/features/Footer/Footer.js:1]: make common Link code for repeating code

File: src/features/Navbar/Navbar.js
Comment (rimple0) [inline src/features/Navbar/Navbar.js:1]: remove unwanted comment code

File: src/features/Banner/Banner.js
Comment (nitindhimangithub) [reply src/features/Banner/Banner.js:1]: done

File: src/features/Footer/Footer.js
Comment (nitindhimangithub) [reply src/features/Footer/Footer.js:1]: done

File: src/features/Navbar/Navbar.js
Comment (nitindhimangithub) [reply src/features/Navbar/Navbar.js:1]: done

**Review by rimple0 — APPROVED** [review]
Approved

**Pull request merged**